README.md

```diff
@@ -1,2 +1,35 @@
-# hw01
-Repository for first CFSS homework assignment
+# HW 01 #
+<hr />
```

Comment:
You shouldn't need to use HTML for the entire document. For specific tasks like image resizing, it is necessary because Markdown doesn't support it. But for simple things like line breaks, Markdown will automatically generate them.